### DISCUSSION TITLE
Re-clustering

### DISCUSSION THREAD

**Discussion by parsanazarian** [satijalab/seurat#9587] [General]
Is it possible to use the SCT assay and Harmony integration for the original integrated Seurat object and then, when subclustering the individual cell types, use the RNA assay for log-normalization and Harmony?

**Comment by samuel-marsh** [comment]
Hi,

Not member of the dev team.  First, as fyi, I will convert this issue to discussion where it is better suited to get answers from community (as opposed to bug reports, errors, etc in Issues).

The basic answer to your question is yes it is possible, but I think you need to think about why you would want to do something like this.  Why use one normalization technique and then switch?  If you are using SCT to help account for differences in sequencing depth these are still going to exist after subsetting so not sure why you want to switch.  So what I would recommend is deeper dive into understanding the differences between the methods and your particular dataset to decide what the best course of action is for you analysis.

Best,
Sam

**Reply by parsanazarian** [reply]
Hello,
Thanks for the response. 
For me, the point of subsetting is to find a new set of variable features within that subset (then PCA, harmony, UMAP, etc), that can separate fine sub-clusters in a given major cell type (e.g, stromal cells). So I thought the old SCT assay won't do the job (since it contains old variable features that are variable between ALL major cell types). So I thought I need a new normalization/finding variable features for the subset, and tried SCTransform() again, creating a new SCT model for each batch present in my subset, with using split() beforehand.
Final UMAP wasn't satisfactory, since expected distinct islands were not present (there was one island)
But doing this process with NormalizeData() on the 'RNA' assay yielded good results with good separations that were also robust.

**Reply by parsanazarian** [reply]
Maybe I can just update the variable features but without new normalization?
